data/changelog.txt

```diff
@@ -1,3 +1,584 @@
+Dinomod NPC Missions
+New Mod Xedra Evolved with new magic classes, monsters, professions and CBMs plus Cyborg professions
+Desert region revival
+Add Megafauna Mod
+[DinoMod] tyrannosaurus bio-operator
+[DinoMod] Nanuqsaurus and burned rebuild
+Aftershock: Add functional jet packs
+Xedra Evolved: New NPCs to meet and rescue
+[DinoMod] player mutations change monster faction behavior
+[DinoMod] stegoceras
+Magiclysm Adding new magic plants
+[DinoMod] compsognathus, eoraptor, amargasaurus bio-operators
+Magiclysm Quest & Lore patch to item distribution
+[DinoMod] mi-go scout tower dino variant
+The Backrooms mod
+Vampire Virus in Xedra Evolved
+Crazy Cataclysm: Adds Powerthirst and Rocket Cans
+Xedra Evolved magic classes Eaters, Dreamers, Inventors and Dreamsmiths
+[Magiclysm] Make Summoning Great Again!
+Implementing more proper alchemy and some another stuff
+Aftershock Space items and smartguns
+Aftershock New CBM and class
+Aftershock: New Monofilament Whip CBM
+My Sweet Cataclysm: Butchering candy monsters gives candy flesh
+Add standard combat test mod
+Nested Crafting Categories
+Add cosmetic variants for mutations
+
+## Balance:
+Mutation System Overhaul
+Made windmills able to grind up dried rice, cooked acorn meal, and roasted cattail rhizome into flour
+Shocker zombies cannot be harvested for bionics in vanilla
+Reduce crafting proficiency penalties based on partial proficiency level
+Updated brewing recipes and fermentation times to be more realistic
+Add stamina cost to archery
+Change Cut Cotton to Patchwork, add Patchwork to Rags
+Stop regenerating creatures from doing so while on fire
+Adds weak points to the armour of most zombies
+Adjusted lockpick crafting
+You can now pry open metal doors with a halligan bar.
+Adjusted how the hoarder morale penalty is calculated with a focus on small pockets and holsters
+Weapon Categories now exist so martial arts can pull by weapon category instead of weapon id
+Massive Martial Arts rebalance
+Massive zombie spawn audits
+Striking techniques and martial arts buffs affect all types of physical damage
+Monster speed and attack speed audits
+Many weapons and armor rebalanced
+Stop letting players climb up via downspouts or ropes when all their arms are broken
+Turret spawns nerfed heavily
+Reduces wetness penalties (and bonuses), as wetness was previously one of the strongest factors affecting morale.
+Cars now more likely to contain tire changing equipment
+Tailoring Audit and rework
+Sublimb rework of armor
+Daily health vs lifestyle system
+Artifacts apply passives in inventory instead of when wielded
+Climate Control CBM (and equivalent effects) now works more consistently, doesn't completely shut down in extreme temperatures, and can stack
+minimum of two zombies to receive suffocation
+Banks deploy cop bots instead of TALON killbots when agitated
+gas masks no longer make you immune to getting sick
+Replace most ASRGs with conventional fuel-powered generators
```

Comment:
```suggestion
Replace most ASRGs with conventional fuel-powered generators
Adds assorted costumes to the objective of the Cosplay mission.
Use find_clear_path always for projectile trajectories, not just when it hits.
Makes using a spear on enemies through fences/bars normally at low levels instead of defaulting to bashing on bars/fence instead.
Add additional sources of patchwork cotton sheets from deconstruction
Mole Crickets and Molerats can now give tetanus with bites

```